### ISSUE TITLE
[gcc7] Cannot compile with fresh Chromebrew install

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#2282]
@lovedheart: I think gcc7 is missing some dependencies.  The gcc package includes the following:
```
  depends_on 'binutils'
  depends_on 'gmp'
  depends_on 'mpfr'
  depends_on 'mpc'
  depends_on 'isl'
  depends_on 'cloog'
  depends_on 'glibc'
```

**Comment by lovedheart** [comment]
@uberhacker But they are in core ?

**Comment by uberhacker** [comment]
Only via gcc which has been replaced by gcc7.

**Comment by uberhacker** [comment]
Fixed in #2283.
